sites/docs/src/content/docs/tutorials/tests_and_test_data/components/03_project_setup.md

```diff
@@ -0,0 +1,392 @@
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "sample": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["id"]
+            },
+            "fastq_1": {
+                "type": "string",
+                "format": "file-path",
+                "exists": true,
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
+                "errorMessage": "FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+            },
+            "fastq_2": {
+                "type": "string",
+                "format": "file-path",
+                "exists": true,
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
+                "errorMessage": "FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+            }
+        },
+        "required": ["sample", "fastq_1"]
+    }
+}
+```
+
+## Project Initialization Steps
+
+### 1. Initialize nf-test (if not present)
+
+```bash
+# Initialize nf-test in the project
+nf-test init
+
+# This creates:
+# - nf-test.config
+# - .nf-test/ directory
+```
+
+### 2. Set Up Test Configuration
+
+Create or verify these key files based on methylseq structure:
+
+#### nf-test.config
+```groovy
+config {
+    testsDir "."
+    workDir System.getenv("NFT_WORKDIR") ?: ".nf-test"
+    configFile "tests/nextflow.config"
+    ignore 'modules/nf-core/**/*', 'subworkflows/nf-core/**/*'
+    profile "test"
+    triggers 'nextflow.config', 'nf-test.config', 'conf/test.config', 'tests/nextflow.config', 'tests/.nftignore'
+    plugins {
+        load "nft-bam@0.5.0"
+        load "nft-utils@0.0.3"
+    }
+}
+```
+
+#### tests/nextflow.config
+```groovy
+params {
+    modules_testdata_base_path   = 's3://ngi-igenomes/testdata/nf-core/modules/'
+    pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/YOUR_PIPELINE/'
+    input       = "${projectDir}/assets/samplesheet.csv"
+    outdir      = 'results'
+}
+
+process {
+    resourceLimits = [
+        cpus: 2,
+        memory: '3.GB',
+        time: '2.h'
+    ]
+}
+
+aws.client.anonymous = true
+manifest { nextflowVersion = '!>=24.10.2' }
+timeline { enabled = false }
+report   { enabled = false }
+trace    { enabled = false }
+dag      { enabled = false }
+```
+
+### 3. Create Test Assets
+
+#### Minimal samplesheet (assets/samplesheet.csv)
+```csv
+sample,fastq_1,fastq_2
+test_sample1,https://github.com/nf-core/test-datasets/raw/YOUR_PIPELINE/testdata/sample1_R1.fastq.gz,https://github.com/nf-core/test-datasets/raw/YOUR_PIPELINE/testdata/sample1_R2.fastq.gz
+test_sample2,https://github.com/nf-core/test-datasets/raw/YOUR_PIPELINE/testdata/sample2.fastq.gz,
+```
+
+### 4. Set Up Profile Configuration
+
+#### conf/test.config
```

Comment:
This should already be present from the hf-core template, shouldn't have to create it